### PR TITLE
Remove rigged static model distinction

### PR DESCRIPTION
**Pull request by ludvigarlebrink**
#131 Removed rigged/static distinction.

### CODE REVIEW

File: src/Engine/Component/Animation.cpp [hunk 1]
```diff
@@ -20,6 +20,6 @@ void Component::Animation::Load(const Json::Value& node) {
     std::string name = node.get("riggedModel", "").asString();
     for (Geometry::Model* model : Hymn().models) {
         if (model->name == name)
-            riggedModel = dynamic_cast<Geometry::RiggedModel*>(model);
+            riggedModel = dynamic_cast<Geometry::Model*>(model);
```
Comment (Chainsawkitten) [inline src/Engine/Component/Animation.cpp]: Casting from Model* to Model* is unnecessary.

File: src/Engine/Geometry/AssetFileHandler.hpp [hunk 2]
```diff
@@ -18,99 +19,103 @@ namespace Geometry {
...
              */
             AssetFileHandler(const char* filepath, Mode mode = READ);
-        
+
             /// Destructor.
             ~AssetFileHandler();
-        
-            /// Open a .asset file.
+
+            /// Open a .wkbf file.
```
Comment (Chainsawkitten) [inline src/Engine/Geometry/AssetFileHandler.hpp]: Aren't they still called .asset?

File: src/Engine/Geometry/AssetFileHandler.hpp [hunk 3]
```diff
@@ -18,99 +19,103 @@ namespace Geometry {
...
+            /// Importing is supported from version.
             const uint16_t SUPPORTED_FROM = 1;
-        
+
             /// Default constructor.
             AssetFileHandler();
-        
-            /// Open a .asset file.
+
+            /// Open a .wkbf file.
```
Comment (Chainsawkitten) [inline src/Engine/Geometry/AssetFileHandler.hpp]: Aren't they still called .asset?

Comment (ludvigarlebrink) [reply src/Engine/Geometry/AssetFileHandler.hpp]: Yes! Fixing it.

**Pull request merged**